bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/OneWireActivator.java

```diff
@@ -22,20 +21,20 @@
  */
 public final class OneWireActivator implements BundleActivator {
 
-	private static Logger logger = LoggerFactory.getLogger(OneWireActivator.class); 
-	
+	private static Logger LOGGER = LoggerFactory.getLogger(OneWireActivator.class);
```

Comment:
please either declare the logger "final" or revert this change. According to http://www.oracle.com/technetwork/java/codeconventions-135099.html only constants should be written in capital letters